src/test/compress_tests.cpp

```diff
@@ -61,4 +62,85 @@ BOOST_AUTO_TEST_CASE(compress_amounts)
         BOOST_CHECK(TestDecode(i));
 }
 
+BOOST_AUTO_TEST_CASE(compress_script_to_ckey_id)
+{
+    // case CKeyID
+    CKey key;
+    key.MakeNewKey(true);
+    CPubKey pubkey = key.GetPubKey();
+
+    CScript script;
+    script.clear();
+    script << OP_DUP << OP_HASH160 << ToByteVector(pubkey.GetID()) << OP_EQUALVERIFY << OP_CHECKSIG;
```

Comment:
nit: This could be put in one line by immediately assigning the script on the declaration: `CScript script = CScript() << OP_DUP ..... << OP_CHECKSIG` (also applies to the other three functions).

Reply:
wouldn't assigning the script on the declaration remove the opportunity to call `.clear()` on it?

Reply:
I don't think that calling `.clear()` is needed here at all, since newly constructed `CScript`s are empty anyway?